portfolio/src/main/webapp/script.js

```diff
@@ -80,44 +80,53 @@ function clearComments() {
 }
 
 /** 
- * Creates paragraph elements and sets the text to comments 
- * pulled from /data page.
- * 
- * TODO: Sperate this method into 2. 
- * 1) Create the p element.
- * 2) Create the popupElement.
+ * This class creates a comment and it's associated popup so it can be added to the comments div.
  */
-function displayComment(comment) {
-  const commentElement = document.createElement("p");
-  commentElement.innerText =
-      `${comment.title} posted on: ${comment.postedDateTime}`;
-
-  const popupElement = document.createElement("span")
-  popupElement.classList.add("popup-content");
-  const commentTitle = document.createElement("h5");
-  commentTitle.innerText = comment.title;
-  const commentText = document.createElement("p");
-  commentText.innerText = comment.text;
-  const commentImg = document.createElement("img");
-  commentImg.src = comment.imageUrl;
-
-  popupElement.appendChild(commentTitle);
-  popupElement.appendChild(commentText);
-
-  if (comment.imageUrl) { 
-    popupElement.appendChild(commentImg);
+class PageComment extends HTMLElement {
+  comment = null;
+
+  constructor(comment) {
+    super();
+    this.comment = comment;
   }
 
-  commentElement.addEventListener("click", () => {
-      popupElement.classList.toggle("show");
-  });
+  connectedCallback() {
+    const {text, postedDateTime, title, imageUrl} = this.comment;
+    if (imageUrl) {
```

Comment:
Could you show me how to do this the quicker way again?

Reply:
```javascript
this.innerHTML = `
        <p id="comment">${text} posted on ${postedDateTime} by ${email}</p>
        <span id="popup" class="popup-content">
          <h5>${title}</h5>
          <p>${text}</p>
          ${imageUrl ? `<img src="${imageUrl}">` : ''}
        </span>
      `;
```